docs/concepts.rst

```diff
@@ -0,0 +1,28 @@
+============
+Concepts
+============
+
+.. _named_table:
+
+Named Tables
+~~~~~~~~~~~~~
+
+These are tables that are of some importance to users and will we persist in a database even after a DAG run is finished. You can create these tables by passing in a ``name`` parameter while creating a ``astro.sql.table.Table`` object.
```

Comment:
Persistent Table instances will persist after the DAG run and are not affected by the :ref:`aql.cleanup` task. Users can still drop them by using :ref:`aql.drop_table`.

Reply:
Added both points.